demos/Run_Stac_Hydra.py

```
@@ -0,0 +1,75 @@
+import os
+
+os.environ["CUDA_VISIBLE_DEVICES"] = "1"  # Use GPU 1
+import stac_mjx
+from pathlib import Path
+from jax import numpy as jp
```

Ruff

`jax.numpy` imported but unused

Remove unused import: `jax.numpy`

(F401)